### DISCUSSION TITLE
List of possible values for the shortForecast object.

### DISCUSSION THREAD

**Discussion by ewleonardspock** [weather-gov/api#293] [Q&A]
Is there anywhere I can find a list of all of the possible values for the shortForecast object returned by the JSON.  I'm trying to associate them with images.  (I'd use the icons you provide, but I need them to be a specific size).

**Comment by vestmon** [comment]
I think this will give you what you need.

https://api.weather.gov/icons

**Comment by ewleonardspock** [comment]
Thanks for that!  Unfortunately, it doesn't match up with what I'm seeing.

Here's the json for my area:  https://api.weather.gov/gridpoints/LWX/65,45/forecast

The current "shortForecast" is "Clear" which isn't in that list, nor is "Sunny" which is the "shortForecast" for tomorrow.

**Comment by scadergit** [comment]
The short forecast is derrived from directives and does not translate to a keyable value itself, but as @vestmon noted the icon url will give you the next best thing--you'll just have to parse it out. In the near future, we're going to replace the icon url with a keyable property to use with any icon set, including the NWS icons if desired.

**Comment by jfbauer432** [comment]
I am not sure how standardized that shortForecast is.  It takes a bit of work, but you can parse the icon field to get what you are after: "skc" and "tsra_hi" are couple of examples from the below URLs.  The full list being listed in that previously mentioned URL.  It would be nice if this was easier .

https://api.weather.gov/icons/land/night/skc?size=medium
https://api.weather.gov/icons/land/day/tsra_hi,60?size=medium
https://api.weather.gov/icons/land/night/tsra_hi,60/tsra_hi,40?size=medium

Take note that the icon name with optional percentage can be repeated a second time.

**Comment by vestmon** [comment]
This may be out of date and does not "Sunny" but is more comprehensive.

https://www.weather.gov/forecast-icons

**Comment by ewleonardspock** [comment]
I'll give that a try.  Thanks!
